CMakeLists.txt

```
@@ -48,19 +48,153 @@ if(POLICY CMP0135)
   cmake_policy(SET CMP0135 NEW)
 endif()
 
+if(WIN32)
+  string(REPLACE "\\" "/" BUILDENV_BASEPATH "${BUILDENV_BASEPATH}")
+  string(REPLACE "\\" "/" CMAKE_TOOLCHAIN_FILE "${CMAKE_TOOLCHAIN_FILE}")
+endif()
```

This mirrors the CMAKE_TOOLCHAIN_FILE path handling, but uses a new variable to not change any cached value. 
```suggestion
file(
  REAL_PATH
  "${CMAKE_TOOLCHAIN_FILE}"
  CLEAN_CMAKE_TOOLCHAIN_FILE
  BASE_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
  EXPAND_TILDE
)
if(DEFINED BUILDENV_NAME)
  file(TO_CMAKE_PATH ${BUILDENV_BASEPATH} BUILDENV_BASEPATH)
  set(MIXXX_VCPKG_ROOT "${BUILDENV_BASEPATH}/${BUILDENV_NAME}")
endif()
```